### DISCUSSION TITLE
Value “ct”:

### DISCUSSION THREAD

**Discussion by Midnight83** [RavenSystem/esp-homekit-devices#1691] [MEPLHAA Scripts]
category value “ct” not saved and changes to accessory after pairing, how to fix the category value after pairing?

**Comment by wahidulislamriyad** [comment]
![0A0BB0B5-AF7D-45E8-B112-C111C230A3B8](https://user-images.githubusercontent.com/32354046/150693917-b266b6ce-d922-493f-9e58-42bc0e0c6e4e.jpeg)

Update the firmware to latest version. You MUST reset HomeKit data when updating json config and remove pairing. “ct” is working perfectly here even after pairing.
